python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -29,6 +29,20 @@ router bgp {{ router_bgp.as }}
 {%     elif router_bgp.bgp.default.ipv4_unicast_transport_ipv6 is arista.avd.defined(false) %}
    no bgp default ipv4-unicast transport ipv6
 {%     endif %}
+{%     if router_bgp.timers is arista.avd.defined %}
+{%         if router_bgp.timers.keepalive_time is arista.avd.defined and router_bgp.timers.hold_time is arista.avd.defined %}
+{%             set timers_bgp_cli = "timers bgp " ~ router_bgp.timers.keepalive_time ~ " " ~ router_bgp.timers.hold_time %}
+{%         elif router_bgp.timers.min_hold_time is arista.avd.defined or router_bgp.timers.send_failure_hold_time is arista.avd.defined %}
+{%             set timers_bgp_cli = "timers bgp" %}
+{%             if router_bgp.timers.min_hold_time is arista.avd.defined %}
+{%                 set timers_bgp_cli = timers_bgp_cli ~ " min-hold-time " ~ router_bgp.timers.min_hold_time %}
+{%             endif %}
+{%             if router_bgp.timers.send_failure_hold_time is arista.avd.defined %}
+{%                 set timers_bgp_cli = timers_bgp_cli ~ " send-failure hold-time " ~ router_bgp.timers.send_failure_hold_time %}
+{%             endif %}
+{%         endif %}
```

Comment:
Why do we render either keepalive+hold or minHold+send-failure? All these settings are ok to go together
```
avd-ci-leaf2(config-s-s14-router-bgp)#timers bgp 1 10 min-hold-time 5 send-failure hold-time 200
avd-ci-leaf2(config-s-s14-router-bgp)#show session-config diffs 
--- system:/running-config
+++ session:/s14-session-config
router bgp 65001
+   timers bgp 1 10 min-hold-time 5 send-failure hold-time 200
avd-ci-leaf2(config-s-s14-router-bgp)#
```